mitiq/zne/inference.py

```diff
@@ -1637,6 +1639,142 @@ def reduce(self) -> float:
 ]
 
 
+class BayesFactory(BatchedFactory):
+    """Factory object implementing a zero-noise extrapolation algorithm based on
+    Bayesian Inference.
+
+    Args:
+        scale_factors: Sequence of noise scale factors at which
+                       expectation values should be measured.
+        shot_list: Optional sequence of integers corresponding to the number
+                   of samples taken for each expectation value. If this
+                   argument is explicitly passed to the factory, it must have
+                   the same length of scale_factors and the executor function
+                   must accept "shots" as a valid keyword argument.
+
+    Raises:
+        ValueError: If data is not consistent with the extrapolation model.
+    Note:
+        RichardsonFactory and LinearFactory are special cases of PolyFactory.
+    """
+
+    @staticmethod
+    def _likelihood(
```

Comment:
Technically I think this is not a properly a _likelihood_ but more a _model_ or an _ansatz_. So I would rename it maybe as `ansatz` or `exp_ansatz`.